pipelines/api-pr-pipeline.yml

```diff
@@ -0,0 +1,169 @@
+                  Write-Host "Registering Fusion PowerShell Repository"
+
+                  if ($null -eq (Get-PSRepository -Name Fusion -ErrorAction SilentlyContinue)) {
+                    Register-PSRepository -Name Fusion -SourceLocation "https://statoil-proview.pkgs.visualstudio.com/5309109e-a734-4064-a84c-fbce45336913/_packaging/Fusion-Public/nuget/v2" -InstallationPolicy Trusted -ErrorAction Stop
+                    Write-Host "Ok"
+                  } else {
+                    Write-Host "Fusion already added as a repository..."
+                  }	
+
+                  Write-Host "Installing FusionPS module for CurrentUser"
+                  Install-Module FusionPS -Repository Fusion -Force -Scope CurrentUser
+
+                  Write-Output (Get-Module -ListAvailable -Name FusionPS)
+
+                  ## Remove repository
+                  if ($null -ne (Get-PSRepository -Name Fusion -ErrorAction SilentlyContinue)) {
+                    Unregister-PSRepository -Name Fusion -ErrorAction Stop		
+                    Write-Host "Ok"
+                  } else {
+                    Write-Host "Was not able to remove the fusion ps repo... This can cause problems on hosted agents."
+                  }	
+
+            - task: AzurePowerShell@4
+              displayName: 'Get secrets'
+              inputs:
+                azureSubscription: $(subscriptionService)
+                ScriptType: 'InlineScript'
+                FailOnStandardError: true
+                azurePowerShellVersion: 'LatestVersion'
+                Inline: |
+                    $secret = Get-AzKeyVaultSecret -VaultName ProView-Shared-Secrets -Name ClientSecret-ProView-CI
+                    Write-Output "##vso[task.setvariable variable=clientSecret;issecret=true]$($secret.SecretValueText)"
+
+            - task: AzurePowerShell@4
+              displayName: 'Provision database'
+              inputs: 
+                azureSubscription: $(subscriptionService)
+                ScriptType: 'InlineScript'
+                FailOnStandardError: true
+                azurePowerShellVersion: 'LatestVersion'
+                Inline: |
+                    Import-Module FusionPS
+
+                    $dbName = "Fusion-Apps-Resources-PR-$(prNumber)-DB"
+
+                    New-FusionSqlDatabaseV2 `
+                      -InfraEnv Test `
+                      -DatabaseName $dbName `
+                      -SourceDatabase Fusion-Apps-Resources-CI-DB `
+                      -Tags @{ "pr" = $(prNumber) }
+
+                    Set-FusionAzSqlDatabaseAccess `
+                      -InfraEnv Test `
+                      -Environment PR `
+                      -DatabaseName $dbName
+                    
+                    $connectionString = "Server=tcp:fusion-test-sqlserver.database.windows.net,1433;Initial Catalog=$($dbName);Persist Security Info=False;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
```

Comment:
Could use the `Get-FusionSqlServerConnectionString -InfraEnv Test -DatabaseName $DatabaseName -Timeout $Timeout` command

Reply:
Yup